public/css/api.main.css

```diff
@@ -4,10 +4,18 @@ body, html{
     background-color: #E0E0E0; 
 }
 
+#settings {
+    -webkit-box-shadow: -1px 2px 16px 3px rgba(0,0,0,0.52);
+    -moz-box-shadow: -1px 2px 16px 3px rgba(0,0,0,0.52);
+    box-shadow: -1px 2px 16px 3px rgba(0,0,0,0.52);
+    margin-top: 65px;
+    height: 100px;
+}
+
 #hero{
```

Comment:
**Code Quality - NitPick**
@andela-araimi there should be space between the id name and the opening curly braces. Fix any other appearance of this in this file.